pkg/pdutil/api_client.go

```diff
@@ -169,6 +169,9 @@ type RegionInfo struct {
 	StartKey    string `json:"start_key"`
 	EndKey      string `json:"end_key"`
 	WrittenKeys uint64 `json:"written_keys"`
+	// NormalizedWeight is the written keys normalized
+	// by the average region written keys of a same table.
+	NormalizedWeight uint64
```

Comment:
remove this field, it's not a part of response

Reply:
sure, I will put it another place.